main/autogen/autogen.cc

```diff
@@ -18,7 +18,7 @@ class AutogenWalk {
     vector<DefinitionRef> nesting;
 
     enum class ScopeType { Class, Block };
-    vector<ast::Send *> ignoring;
+    vector<ast::ExpressionPtr::tagged_storage> ignoring;
```

Comment:
Is `ExpressionPtr::tagged_storage` API that we'd prefer not to expose?

Reply:
It's not my favorite API in the world, and it's not great to have another use of it in the tree.  I think manually doing some kind of pointer tagging is the only alternative (see other comment in this file).